Unstructured-IO/unstructured-api-tools
# feat: read FastAPI metadata from the config file

### Summary

Updates the API generation code to allow for FastAPI metadata to be read from the config file.

### Testing

- Install this branch in an venv for `pipeline-oer`
- Update the `pipeline-oer` config to the following

```
name: oer
version: 0.0.1
description: Unstructured OER Pipeline
long_description: |
  The Unstructured OER pipeline parses rater/senior rater comments
  and check boxes from Army OER. Use the /raters endpoint to process
  OER PDF files.
```

- Run `make generate-api`
- Run `make run-web-app`
- The fast API docs at http://localhost:8000/docs should look like the following

<img width="1493" alt="image" src="https://user-images.githubusercontent.com/1635179/205092754-0f26c966-2b75-4523-ad29-6652ac904773.png">
